### PR TITLE
support using default queue

### PR DESCRIPTION
**Pull request by tornado404**
### PR types
Function optimization

### PR changes
Others

### Describe
support using default queue

### CODE REVIEW

File: pkg/apiserver/controller/job/create.go [hunk 1]
```diff
@@ -179,7 +182,11 @@ func validateMembersResource(ctx *logger.RequestContext, request *CreateJobInfo)
 		sumResource.Add(memberRes)
 	}
 	// validate queue and total-member-resource
-	if !sumResource.LessEqual(request.SchedulingPolicy.MaxResources) {
+	// if EnvSkipRV is set, it will pass the validation for 0-Capability volcano queue
+	skipResValidate := os.Getenv(EnvSkipResourceValidate) != "" &&
```
Comment (D0m021ng) [inline pkg/apiserver/controller/job/create.go]: Add a variable `skipResValidate` and get from env.

File: pkg/apiserver/controller/job/create.go [hunk 2]
```diff
@@ -247,46 +254,24 @@ func validateJobMembers(ctx *logger.RequestContext, request *CreateJobInfo) erro
 		return err
 	}
 
+	// validate resource in members
+	if err := validateMembersResource(ctx, request); err != nil {
+		ctx.Logging().Errorf("validate members role failed, err: %v", err)
```
Comment (D0m021ng) [inline pkg/apiserver/controller/job/create.go]: Error message is `validate members resoruces`

File: pkg/apiserver/controller/job/create.go [hunk 3]
```diff
@@ -51,6 +56,11 @@ type CreateJobInfo struct {
 	ExtensionTemplate map[string]interface{} `json:"extensionTemplate,omitempty"`
 }
 
+func init() {
+	IsSkipResourceValidate = os.Getenv(EnvSkipResourceValidate) != ""
+	log.Debugf("IsSkipResourceValidate: %v\n", IsSkipResourceValidate)
```
Comment (D0m021ng) [inline pkg/apiserver/controller/job/create.go]: Use `fmt.Printf`

**Review by D0m021ng — APPROVED** [review]

File: pkg/apiserver/controller/job/create.go [hunk 4]
```diff
@@ -51,6 +56,11 @@ type CreateJobInfo struct {
 	ExtensionTemplate map[string]interface{} `json:"extensionTemplate,omitempty"`
 }
 
+func init() {
+	IsSkipResourceValidate = os.Getenv(EnvSkipResourceValidate) != ""
+	fmt.Printf("IsSkipResourceValidate: %v\n", IsSkipResourceValidate)
```
Comment (luoyuedong) [inline pkg/apiserver/controller/job/create.go:61]: 为啥要这样打印下

**Review by luoyuedong — APPROVED** [review]

**Pull request merged**